.eslintrc

```diff
@@ -7,6 +7,7 @@
   "rules": {
     "flowtype/no-weak-types": 0,
     "flowtype/require-parameter-type": 0,
-    "flowtype/require-return-type": 0
+    "flowtype/require-return-type": 0,
+    "id-length": 0
```

Comment:
If I disable an eslint rule in test but not for build, then a rule could be violated in test and would not notify you but would cause the build to fail.

Is this what we want to do?